### PR TITLE
Upgrade to 4.44

### PR DESCRIPTION
**Pull request by SameerK-MSFT**

### CODE REVIEW

File: 2-With-broker/UserDetailsClient/UserDetailsClient/App.cs [hunk 1]
```diff
@@ -29,10 +29,6 @@ public class App : Application
 
         public App()
         {
-            PCA = PublicClientApplicationBuilder.Create(ClientID)
```
Comment (neha-bhargava) [inline 2-With-broker/UserDetailsClient/UserDetailsClient/App.cs:32]: Why is this removed?

Comment (SameerK-MSFT) [reply 2-With-broker/UserDetailsClient/UserDetailsClient/App.cs:32]: This is created only once and this creation is with no broker. And It is created when needed.

**Review by gladjohn — APPROVED** [review]
